### ISSUE TITLE
Comment will be postet again after page refresh

### ISSUE THREAD

**Issue by Endogen** [getgrav/grav-plugin-comments#63]
1. Comment on a blog page
2. Refresh the page
3. Comment will be posted again
4. Refresh page
5. Comment will be posted again (3 times now)

**Comment by rafalzdeb** [comment]
Experiencing exactly the same behaviour.

**Comment by ulab** [comment]
I thought the "Refresh prevention" option of the Form module would help with this, but it looks like it does not....

**Comment by wojasmikas** [comment]
Hej, you need to edit comments.php and add this line:

`                header("location: " .$this->grav['uri']->url );`

in this function:

`    public function onFormProcessed(Event $event)`

before `break;`  (probably line 247).

I found the solution right here:

https://bytes.com/topic/php/answers/12629-destroy-_post-vars-after-use-possible?fbclid=IwAR2R5kC-94rtE4hgLiarXtj7zYJU4TLdTRQ49xgTBsji44ABPWnVIIhVMEI

Good luck

**Comment by rhukster** [comment]
I can't recrete this. My test comment form is the stock one, has a `reset: true` as the last action, but has no `refresh_prevention: true` in the form definition.

**Comment by ulab** [comment]
I have just done a basic Grav v1.6.27 - Admin v1.9.16 installation and added the Comments v1.2.7 plugin.

I added the plugins include line to `user/themes/quark/templates/default.html.twig` , created `user/config/plugins` and edited `user/config/plugins/comments.yaml` to include the example `user/plugins/comments/comments.yaml` and changed the following:

```
enable_on_routes:
  - '/'

disable_on_routes:
```

I've also set `From address`and `To address` in the Email plugin configuration.

Now I have a comment form on the homepage. I can fill it out, send the comment and when I refresh the page the browser asks me if I wanted to resend the data. 

When I do that, it adds the comment again.

If I enable `Refresh prevention` in the form plugin and load the page again (not reload, but from the browser bar), I can repeat the exact same procedure.
